### PR TITLE
fix duplicate notification of star event

### PR DESCRIPTION
**Pull request by kenautomus**

### CODE REVIEW

File: config/jsons/github-events.json [hunk 1]
```diff
@@ -113,10 +113,10 @@
   },
   "star": {
     "created": true,
-    "deleted": false
+    "deleted": true
   },
   "watch": {
-    "started": true
+    "started": false
```
Comment (tanhongit) [inline config/jsons/github-events.json:119]: @kenautomus Why do we need to change the condition of the watch event?

Comment (kenautomus) [reply config/jsons/github-events.json:119]: @tanhongit  the started action in this event is also to send the star notification.  

I realized that when there is a new star, there will be 2 events sent at the same time: watch with start action and star with create action

that is the cause of duplication

**Comment by tanhongit** [comment]
PR is related at https://github.com/cslant/laravel-telegram-git-notifier/pull/62

File: config/jsons/github-events.json (hunk 1 above)
Comment (tanhongit) [reply config/jsons/github-events.json:119]: @kenautomus I understood, thanks!

**Pull request merged**